### DISCUSSION TITLE
Jupyter interactive cannot use via remote ssh it shown chunkLoadError #4698 My solution to this issue (can vary by user and just share how I solved it)

### DISCUSSION THREAD

**Discussion by wonjunchoi-arc** [microsoft/vscode-jupyter#14543] [Questions and Answers]

**Comment by DonJayamanne** [comment]
Sorry you are running into this issues
This problem was fixed today in the latest pylance release
This was caused by another extension, not Jupyter 

Please update your pylance extension to the latest version
